app/views/layouts/application.html.erb

```diff
@@ -44,14 +44,6 @@
   </div><!-- /.container-fluid -->
 </nav>
 
-<!-- Flash Messages -->
-<% if notice.present? %>
-  <p class="alert alert-info"><%= notice %></p>
-<% end %>
-<% if alert.present? %>
-  <p class="alert alert-danger"><%= alert %></p>
-<% end %>
```

Comment:
Will this prevent other flash messages from appearing? Such as "Signed in successfully"